MoneyHoney/redux/store.jsx

```diff
@@ -0,0 +1,5 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import reducer from './reducer';
+
+export default createStore(reducer, applyMiddleware(thunk));
```

Comment:
we should use `combineReducer` otherwise our reducer will be too big

Reply:
applied combineRducer